src/Indicator.vala

```diff
@@ -210,6 +211,41 @@ public class Notifications.Indicator : Wingpanel.Indicator {
             warning ("Failed to open notifications settings: %s", e.message);
         }
     }
+
+    private void update_tooltip () {
+        uint number_of_notifications = Session.get_instance ().get_session_notifications ().length ();
+        int number_of_apps = nlist.app_entries.size;
+
+        string description;
+        string accel_label;
+
+        if (notify_settings.get_boolean ("do-not-disturb")) {
+            accel_label = _("Middle-click to disable Do Not Disturb");
+        } else {
+            accel_label = _("Middle-click to enable Do Not Disturb");
+        }
+
+        accel_label = Granite.TOOLTIP_SECONDARY_TEXT_MARKUP.printf (accel_label);
+
+        switch (number_of_notifications) {
+            case 0:
+                description = _("No notifications");
+                break;
+            case 1:
+                description = _("1 notification");
+                break;
```

Comment:
Just out of curiously, but is there any reason you set this condition? I feel you can merge this condition with the `default` and say: "1 notification from 1 app".

If we merge this condition we have only 2 conditions in this `switch` statement so we may want to use `if` instead.

Reply:
Because 1 notification from 1 app would sound a little weird (in English I guess). If you have only 1 notification, it's only possible that it is from 1 app so putting 1 notification from 1 app would make the "from 1 app" part sound a little redundant (in English) and so I didn't think I would need to show it to the user but yes I do agree it would simplify this into 2 conditions if we combine them.

Reply:
Ah you're right, it's redundant indeed… simplicity in the code or in the copy :thinking: